addons/project/wizard/project_report_wizard.py

```diff
@@ -0,0 +1,29 @@
+# -*- coding: utf-8 -*-
+# Part of Odoo. See LICENSE file for full copyright and licensing details.
+
+from odoo import fields, models
+
+
+class ProjectReportWizard(models.TransientModel):
+    _name = 'project.report.wizard'
+    _description = 'Project Report Wizard'
+
+    project_id = fields.Many2one('project.project', required=True)
+    report_type = fields.Selection([
+        ('burndown', 'Burndown Chart'), ('project_update', 'Project Update')
+    ], string='Report Type', required=True, default='burndown')
+
+    def action_burndown_report_view(self):
+        action = self.env['ir.actions.act_window']._for_xml_id('project.action_project_task_burndown_chart_report')
+        action['context'] = {'search_default_project_id': self.project_id.id}
+        return action
+
+    def action_project_update_report_view(self):
+        action = self.env['ir.actions.act_window']._for_xml_id('project.project_update_all_action')
+        action.update({
+            'context': {
+                'active_id': self.project_id.id,
+                'search_default_project_id': self.project_id.id,
+            }
+        })
```

Comment:
In order not to overwrite an eventual context in the action, I would rather do this otherwise you will lose your `default_report_type` context key:
```suggestion
        action['context'] = dict(literal_eval(action.get('context'), {}), active_id=self.project_id.id, search_default_project_id=self.project_id.id)
```

Reply:
@IT-Ideas , here we have active_id as a variable(without quote like: `"{'search_default_project_id': active_id}"`) in context so while evaluating the `literal_eval(action.get('context')` it will give us the traceback as `ValueError: malformed node or string: <ast.Name object at 0x112cf0250>` due to active_id. 

so we have to use the safe_eval like:
```suggestion 
action['context'] = dict(safe_eval(action['context'].strip(), {'active_id': self.project_id.id}), search_default_project_id=self.project_id.id)
```

what do you think?

Reply:
@IT-Ideas , here we have active_id as a variable(without quote like: `"{'search_default_project_id': active_id}"`) in context so while evaluating the `literal_eval(action.get('context')` it will give us the traceback as `ValueError: malformed node or string: <ast.Name object at 0x112cf0250>` due to active_id. 

so we have to use the safe_eval like:
```suggestion 
action['context'] = dict(safe_eval(action['context'].strip(), {'active_id': self.project_id.id}), search_default_project_id=self.project_id.id)
```

what do you think?